spark/src/test/java/org/apache/iceberg/spark/data/AvroDataTest.java

```diff
@@ -40,6 +42,8 @@
 
   protected abstract void writeAndValidate(Schema schema) throws IOException;
 
+  protected abstract SparkSession getSparkSession();
```

Comment:
Do we need this? Just wondering if we can just do SparkSession.getActive or just change our SparkSession creation code in the testbase to always set the handle TS.withoutZone flag to true

Reply:
Like I see we use SparkSession.builder().master("local[2]").getOrCreate(); in a bunch of places, we could just switch that to a utility method that always sets the handle TZ prop

Reply:
Agree, I have removed `getSparkSession()` abstract method, and just used `SparkSession.getActive`